### PR TITLE
Extend logging utility

### PR DESCRIPTION
**Pull request by armadilloz**
Added ability to use and extend Logging group commands directly, in order to allow custom command groups.
Added examples and tests - will move to common utils.

### CODE REVIEW

File: src/main/java/frc/robot/utils/logging/SequentialLoggingCommand.java [hunk 1]
```diff
@@ -29,6 +47,11 @@ public String toString() {
         return getFullyQualifiedName();
     }
 
+    // For testing
+    public List<LoggingCommand> getLoggingCommands() {
```
Comment (noahheller) [inline src/main/java/frc/robot/utils/logging/SequentialLoggingCommand.java]: this says "For testing" do we still need it?

Comment (armadilloz) [reply src/main/java/frc/robot/utils/logging/SequentialLoggingCommand.java]: Yes - I intend to add the tests I wrote to the common repo

Comment (armadilloz) [reply src/main/java/frc/robot/utils/logging/SequentialLoggingCommand.java]: Here are the tests for the CommandUtil: https://github.com/FRC4048/java-common/pull/11

**Review by MatthewSpofford — DISMISSED** [review]

**Comment by noahheller** [comment]
Can you add a deadline command option?

**Comment by noahheller** [comment]
Can we merge this asap? Would come in very handing.

**Review by batchen1 — APPROVED** [review]

**Pull request merged**